pyramid_zipkin/tween.py

```diff
@@ -192,23 +192,33 @@ def tween(request: Request) -> Response:
             tween_kwargs['firehose_handler'] = zipkin_settings.firehose_handler
 
         with tracer.zipkin_span(**tween_kwargs) as zipkin_context:
-            response = handler(request)
-            if zipkin_settings.use_pattern_as_span_name and request.matched_route:
-                zipkin_context.override_span_name('{} {}'.format(
-                    request.method,
-                    request.matched_route.pattern,
-                ))
-            zipkin_context.update_binary_annotations(
-                get_binary_annotations(request, response),
-            )
-
-            if zipkin_settings.post_handler_hook:
-                zipkin_settings.post_handler_hook(
-                    request,
-                    response,
-                    zipkin_context
+            response = None
+            try:
+                response = handler(request)
+            except Exception as e:
+                zipkin_context.update_binary_annotations({
+                    'error_type': type(e).__name__,
```

Comment:
to ease any future efforts on adapting to opentelemetry's semantic conventions can we use `error.type` as the annotation instead? 
Source -> [http-spans](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-spans.md#common-attributes)

Reply:
good call out! made the change